mozregression/cli.py

```diff
@@ -265,6 +265,12 @@ def create_parser(defaults):
                              " release using a specific release number, and"
                              " a nightly using a specific buildid")
 
+    parser.add_argument('-P', '--process-output', choices=('0', '1'),
+                        default=defaults['process-output'],
+                        help=("Manage process output logging. 1 to enable, 0"
```

Comment:
Yes. That would be weird to me, cmdline options depending on the config file... What if I just replace "1" and "0" possible values (yeah, that is quite horrible now that I see that again) to "stdout" and "none" ? There would be some space in case we want to add another new action, and "-P=stout" looks ok to me.
Will ?